### DISCUSSION TITLE
Snap and Read

### DISCUSSION THREAD

**Discussion by Asriel249** [3kh0/ext-remover#1183] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Was browsing ext-remover.net and tried to execute a bookmarklet through snap and read. The automatic option didn’t work(said it wasn’t installed) and when I try manual it doesn’t work either, is this method a way to unblock JavaScript:// pages or does it do something else I’m not getting?

**Comment by solanuscasey** [comment]
snap and read? as far as I know if javascript:// is blocked in [chrome://policy ](url)then you can't unblock it. I could be wrong though.

**Comment by AshtonDavies** [comment, marked as answer]
There might be bugs with the integration on a newer Snap&Read update. Sorry, I am very slow to fix stuff like this. I might not fix it, idk.

You should try the refresh button a few times if you haven't already.
